kratos/python/add_constraint_to_python.cpp

```diff
@@ -0,0 +1,62 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Aditya Ghantasala
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/define_python.h"
+#include "add_constraint_to_python.h"
+#include "includes/master_slave_constraint.h"
+
+namespace Kratos
+{
+namespace Python
+{
+using namespace pybind11;
```

Comment:
I would suggest to make the interface without using namespace

Reply:
Done !